app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -31,7 +31,8 @@ def accept; end
 
   def accept_assignment
     create_group_assignment_repo do
-      if group_assignment_repo.starter_code_repo_id
+      if repo_setup_enabled? && group_assignment_repo.starter_code_repo_id
+        RepoSetupJob.set(wait: 1.hour).perform_later(GroupAssignmentRepo.name, group_assignment_repo.id)
```

Comment:
Instead of passing a type and an id, you can actually just pass an ActiveRecord object to an ActiveJob and it'll handle serialization and deserialization for you. Then in your job, you can check the type using `#is_a?`